### PR TITLE
Show Utbetalingsinfo for Fagomrade SPREF(Arbeidsgiver)

### PR DESCRIPTION
**Pull request by janzh**

### CODE REVIEW

**Comment by janzh** [comment]
Visning:
image

File: src/components/vedtak/VedtakOppsummering.tsx [hunk 1]
```diff
@@ -12,12 +12,21 @@ import {
 } from 'nav-frontend-typografi';
 import { restdatoTildato } from '../../utils/datoUtils';
 import { VedtakDTO } from '../../reducers/vedtak';
-import { estimertMaksdato } from '../../utils/vedtakUtils';
+import {
+    estimertMaksdato,
+    refusjonTilUtbetalingsbelopBrutto,
+    refusjonTilUtbetalingsdager, VedtakFagomrade
```
Comment (JMLindseth) [inline src/components/vedtak/VedtakOppsummering.tsx:18]: \n

Comment (janzh) [reply src/components/vedtak/VedtakOppsummering.tsx]: Fikset!

**Review by cyanboy — DISMISSED** [review]

File: src/components/vedtak/VedtakOppsummering.tsx [hunk 2]
```diff
@@ -12,12 +12,22 @@ import {
 } from 'nav-frontend-typografi';
 import { restdatoTildato } from '../../utils/datoUtils';
 import { VedtakDTO } from '../../reducers/vedtak';
-import { estimertMaksdato } from '../../utils/vedtakUtils';
+import {
+    estimertMaksdato,
+    refusjonTilUtbetalingsbelopBrutto,
+    refusjonTilUtbetalingsdager,
+    VedtakFagomrade
```
Comment (JMLindseth) [inline src/components/vedtak/VedtakOppsummering.tsx:19]: `,`

Comment (janzh) [reply src/components/vedtak/VedtakOppsummering.tsx]: Fikset.

**Pull request merged**